src/emulator/storage/index.ts

```diff
@@ -86,8 +82,12 @@ export class StorageEmulator implements EmulatorInstance {
     // No-op
   }
 
-  async setRules(rules: SourceFile): Promise<StorageRulesIssues> {
-    return this._rulesManager.setSourceFile(rules);
+  getRules(resource: string): StorageRulesetInstance | undefined {
+    return this._rulesManager.getRuleset(resource);
+  }
+
+  async setRules(rules: RulesType, resource: string): Promise<StorageRulesIssues> {
```

Comment:
we already have a pattern of accessing StorageEmulator internals from within `server.ts` so I'm not sure we need to wrap the rules manager's interface here

Reply:
Ok, I can expose the rules manager instead for the sake of consistency. Curious, is there any benefit to exposing the internals?